_data/simple-icons.json

```diff
@@ -85,6 +100,11 @@
             "hex": "FF5A5F",
             "source": "https://www.airbnb.com"
         },
+        {
+            "title": "Alexa",
```

Comment:
Maybe ["Amazon Alexa"](https://developer.amazon.com/docs/alexa-voice-service/marketing-and-branding-guidelines.html)? They seem to use "amazon alexa", "Amazon Alexa" "Alexa" and "alexa" - I think using the specific version would help differentiate from e.g. [the *other* Alexa by Amazon](https://www.alexa.com/). Thoughts @ericcornelissen @davidklebanoff ?